docs/pages/database-access/guides/rds.mdx

```diff
@@ -355,12 +462,16 @@ $ tsh db ls
 # aurora-mysql                   Aurora cluster in us-west-1                   ...
 # aurora-mysql-custom-myendpoint Aurora cluster in us-west-1 (custom endpoint) ...
 # aurora-mysql-reader            Aurora cluster in us-west-1 (reader endpoint) ...
+# rds-proxy                      RDS proxy in us-west-1                        ...
+# rds-proxy-read-endpoint        RDS proxy endpoint in us-west-1               ...
 ```
 
 <Admonition type="note" title="Note">
   Primary, reader, and custom endpoints of Aurora clusters have names of
   `<cluster-id>`, `<cluster-id>-reader`, and
   `<cluster-id>-custom-<endpoint-name>` respectively.
+  RDS proxies and proxy endpoints have names of `<proxy-name>` and
```

Comment:
```suggestion
 
RDS proxies and proxy endpoints have names of `<proxy-name>` and
```

Reply:
Also, "have names of" is kind of vague to me. I think this note would be much clearer as a table or list. An example:

```md
In RDS deployments, endpoint names have the following formats:

|Service|Primary|Reader|Custom|
|:---|:---|:---|:---|
|Aurora|<cluster-id>|<cluster-id>-reader|<cluster_id>-custom-<endpoint-name>|
```
(and so on)

Reply:
added a new table, please review, thanks